api/testworkflows/v1/step_types.go

```diff
@@ -213,6 +214,9 @@ type StepExecuteWorkflow struct {
 
 	// configuration to pass for the workflow
 	Config map[string]intstr.IntOrString `json:"config,omitempty" expr:"template"`
+
+	// selector is used to identify a group of test workflows based on their metadata labels
+	Selector *metav1.LabelSelector `json:"selector,omitempty" expr:"include"`
```

Comment:
Probably better to move it under `name` (as the CRDs in i.e. "Definition" view are generated in the order from these structs)